pandas/core/window.py

```diff
@@ -963,8 +963,18 @@ def count(self):
         not passed. In the future `raw` will default to False.
 
         .. versionadded:: 0.23.0
+    *args, **kwargs :
+        Arguments and keyword arguments to be passed into func.
 
-    \*args and \*\*kwargs are passed to the function""")
+    Returns
+    -------
+    Same type as input
```

Comment:
Need a period here, no?

Reply:
that's actually the type, the right format should be something like:
```
Returns
-------
Series
    Some description ending with a period.
```

@LJArendse if you can please update.

Reply:
Will do, thanks @datapythonista & @WillAyd

Reply:
@datapythonista & @WillAyd 
Done with commit: f20e9515414be38458efc147c4e0e6f420069696